cmd/customers/main.go

```diff
@@ -76,6 +84,21 @@ func main() {
 		log.Fatal().Err(err).Send()
 	}
 
+	mux := http.NewServeMux()
+	mux.HandleFunc("GET /metrics", func(w http.ResponseWriter, r *http.Request) {
+		metrics.WritePrometheus(w, false)
+	})
+	s := http.Server{
+		Addr:              cfg.addr,
+		Handler:           mux,
+		ReadHeaderTimeout: ReadHeaderTimeout,
+	}
+	go func() {
+		err := s.ListenAndServe()
+		if err != nil {
+			log.Fatal().Err(err).Send()
+		}
+	}()
```

Comment:
recommendation: you could implement your own server with this endpoint included by default and reuse it in all your services